### ISSUE TITLE
After updating Calibre image from 4.23 to 5.3.0 unable to start up calibre

### ISSUE THREAD

**Issue by ernholm** [linuxserver/docker-calibre#37]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

I have a fully working Calibre setup using the following image 
Linuxserver.io version:- v4.23.0-ls75 Build-date:- 2020-08-25T09:24:42+00:00

Starting/Stopping/using works as expected.

If I pull the the latest image (V5.3.0-ls83) and start up the application again I end up with a trace-back and the start-up enters an loop and never successfully starts up so I need to revert back to previous image using docker load.

------------------------------

## Expected Behavior

I expect the application to start up for this version in the same way as it did for earlier version
## Current Behavior

I get into endless loop and service does not start up, see docker logs for details.

## Steps to Reproduce

1. sudo docker-compose down
2. sudo docker-compose pull
3. sudo docker-compose up
4. 

## Environment
**OS:** Ubuntu 20.04 LTS Server
**CPU architecture:** x86_64
**How docker service was installed:**

Official docker repo

## Command used to create docker container (run/create/compose/screenshot)

Below is docker_compose.yaml (with user/pass redacted)
---
version: "2"
services:
  calibre:
    image: linuxserver/calibre
    container_name: calibre
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Oslo
      - GUAC_USER=REDACTED #optional
      - GUAC_PASS=REDACTED #optional
    volumes:
      - ./conf:/config
    ports:
      - 6080:8080
      - 6081:8081
    restart: unless-stopped

## Docker logs

 sudo docker-compose up
Recreating calibre ... done
Attaching to calibre
calibre    | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
calibre    | [s6-init] ensuring user provided files have correct perms...exited 0.
calibre    | [fix-attrs.d] applying ownership & permissions fixes...
calibre    | [fix-attrs.d] done.
calibre    | [cont-init.d] executing container initialization scripts...
calibre    | [cont-init.d] 01-envfile: executing... 
calibre    | [cont-init.d] 01-envfile: exited 0.
calibre    | [cont-init.d] 10-adduser: executing... 
calibre    | 
calibre    | -------------------------------------
calibre    |           _         ()
calibre    |          | |  ___   _    __
calibre    |          | | / __| | |  /  \ 
calibre    |          | | \__ \ | | | () |
calibre    |          |_| |___/ |_|  \__/
calibre    | 
calibre    | 
calibre    | Brought to you by linuxserver.io
calibre    | -------------------------------------
calibre    | 
calibre    | To support LSIO projects visit:
calibre    | https://www.linuxserver.io/donate/
calibre    | -------------------------------------
calibre    | GID/UID
calibre    | -------------------------------------
calibre    | 
calibre    | User uid:    1000
calibre    | User gid:    1000
calibre    | -------------------------------------
calibre    | 
calibre    | [cont-init.d] 10-adduser: exited 0.
calibre    | [cont-init.d] 11-moduser: executing... 
calibre    | [cont-init.d] 11-moduser: exited 0.
calibre    | [cont-init.d] 12-prep_xrdp: executing... 
calibre    | [cont-init.d] 12-prep_xrdp: exited 0.
calibre    | [cont-init.d] 13-update_app_name: executing... 
calibre    | [cont-init.d] 13-update_app_name: exited 0.
calibre    | [cont-init.d] 14-configure_openbox: executing... 
calibre    | [cont-init.d] 14-configure_openbox: exited 0.
calibre    | [cont-init.d] 30-update_webapp_context: executing... 
calibre    | [cont-init.d] 30-update_webapp_context: exited 0.
calibre    | [cont-init.d] 35-update_guac_creds: executing... 
calibre    | [cont-init.d] 35-update_guac_creds: exited 0.
calibre    | [cont-init.d] 50-config: executing... 
calibre    | [cont-init.d] 50-config: exited 0.
calibre    | [cont-init.d] 99-custom-scripts: executing... 
calibre    | [custom-init] no custom files found exiting...
calibre    | [cont-init.d] 99-custom-scripts: exited 0.
calibre    | [cont-init.d] done.
calibre    | [services.d] starting services
calibre    | [services.d] done.
calibre    | 
calibre    | X.Org X Server 1.19.6
calibre    | Release Date: 2017-12-20
calibre    | X Protocol Version 11, Revision 0
calibre    | Build Operating System: Linux 4.15.0-115-generic x86_64 Ubuntu
calibre    | Current Operating System: Linux 80a2760ba089 5.4.0-48-generic #52-Ubuntu SMP Thu Sep 10 10:58:49 UTC 2020 x86_64
calibre    | Kernel command line: BOOT_IMAGE=/vmlinuz-5.4.0-48-generic root=/dev/mapper/ubuntu--vg-ubuntu--lv ro
calibre    | Build Date: 04 September 2020  03:34:39PM
calibre    | xorg-server 2:1.19.6-1ubuntu4.6 (For technical support please see http://www.ubuntu.com/support) 
calibre    | Current version of pixman: 0.34.0
calibre    | 	Before reporting problems, check http://wiki.x.org
calibre    | 	to make sure that you have the latest version.
calibre    | Markers: (--) probed, (**) from config file, (==) default setting,
calibre    | 	(++) from command line, (!!) notice, (II) informational,
calibre    | 	(WW) warning, (EE) error, (NI) not implemented, (??) unknown.
calibre    | (==) Log file: "/var/log/Xorg.pid-429.log", Time: Sat Oct 17 11:52:28 2020
calibre    | (++) Using config file: "/etc/X11/xrdp/xorg.conf"
calibre    | (==) Using system config directory "/usr/share/X11/xorg.conf.d"
calibre    | guacd[426]: INFO:	Guacamole proxy daemon (guacd) version 0.9.14 started
calibre    | guacd[426]: INFO:	Listening on host 127.0.0.1, port 4822
calibre    | xorgxrdpSetup:
calibre    | xrdpdevSetup:
calibre    | rdpmousePlug:
calibre    | rdpkeybPlug:
calibre    | rdpIdentify:
calibre    | rdpDriverFunc: op 10
calibre    | rdpProbe:
calibre    | rdpPreInit:
calibre    | rdpScreenInit: virtualX 800 virtualY 600 rgbBits 8 depth 24
calibre    | rdpScreenInit: pfbMemory bytes 1920000
calibre    | rdpScreenInit: pfbMemory 0x7fc7e1f31010
calibre    | rdpSimdInit: assigning yuv functions
calibre    | rdpSimdInit: cpuid ax 1 cx 0 return ax 0x00040651 bx 0x00100800 cx 0x7ffafbbf dx 0xbfebfbff
calibre    | rdpSimdInit: sse2 amd64 yuv functions assigned
calibre    | rdpXvInit: depth 24
calibre    | rdpClientConInit: kill disconnected [0] timeout [0] sec
calibre    | 
calibre    | rdpScreenInit: out
calibre    | rdpmousePreInit: drv 0x55e6ea5fa990 info 0x55e6ea771730, flags 0x0
calibre    | rdpmouseControl: what 0
calibre    | rdpmouseDeviceInit:
calibre    | rdpmouseCtrl:
calibre    | rdpRegisterInputCallback: type 1 proc 0x7fc7e230ac60
calibre    | rdpmouseControl: what 1
calibre    | rdpmouseDeviceOn:
calibre    | rdpkeybPreInit: drv 0x55e6ea601550 info 0x55e6ea8e2350, flags 0x0
calibre    | rdpkeybControl: what 0
calibre    | rdpkeybDeviceInit:
calibre    | rdpkeybChangeKeyboardControl:
calibre    | rdpkeybChangeKeyboardControl: autoRepeat on
calibre    | rdpRegisterInputCallback: type 0 proc 0x7fc7e2107530
calibre    | rdpkeybControl: what 1
calibre    | rdpkeybDeviceOn:
calibre    | rdpSaveScreen:
calibre    | rdpDeferredRandR:
calibre    | rdpResizeSession: width 1024 height 768
calibre    |   calling RRScreenSizeSet
calibre    | rdpRRScreenSetSize: width 1024 height 768 mmWidth 271 mmHeight 203
calibre    | rdpRRGetInfo:
calibre    |   screen resized to 1024x768
calibre    |   RRScreenSizeSet ok 1
calibre    | Warning: Cannot convert string "-*-helvetica-bold-r-normal--*-120-*-*-*-*-iso8859-1" to type FontStruct
calibre    | Warning: Cannot convert string "-*-courier-medium-r-normal--*-120-*-*-*-*-iso8859-1" to type FontStruct
calibre    | rdpRRGetInfo:
calibre    | Obt-Message: Xinerama extension is not present on the server
calibre    | rdpInDeferredUpdateCallback:
calibre    | rdpkeybChangeKeyboardControl:
calibre    | rdpkeybChangeKeyboardControl: autoRepeat off
calibre    | Openbox-Message: Unable to find a valid menu file "/var/lib/openbox/debian-menu.xml"
calibre    | /usr/bin/env: 'python': No such file or directory
calibre    | guacd[426]: INFO:	Guacamole connection closed during handshake
calibre    | rdpRRGetInfo:
calibre    | QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-abc'
calibre    | NOTE: Picked up JDK_JAVA_OPTIONS:  --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens=java.rmi/sun.rmi.transport=ALL-UNNAMED
calibre    | 17-Oct-2020 13:52:30.134 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with JAR file [/usr/share/tomcat8/lib/commons-dbcp.jar], exists: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.145 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with JAR file [/usr/share/tomcat8/lib/commons-pool.jar], exists: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.149 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/var/lib/tomcat8/common/classes], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.150 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/var/lib/tomcat8/common], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.151 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/usr/share/tomcat8/common/classes], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.152 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/usr/share/tomcat8/common], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.155 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/var/lib/tomcat8/server/classes], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.155 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/var/lib/tomcat8/server], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.156 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/usr/share/tomcat8/server/classes], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.156 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/usr/share/tomcat8/server], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.158 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/var/lib/tomcat8/shared/classes], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.158 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/var/lib/tomcat8/shared], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.158 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/usr/share/tomcat8/shared/classes], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.159 WARNING [main] org.apache.catalina.startup.ClassLoaderFactory.validateFile Problem with directory [/usr/share/tomcat8/shared], exists: [false], isDirectory: [false], canRead: [false]
calibre    | 17-Oct-2020 13:52:30.268 SEVERE [main] org.apache.catalina.startup.Catalina.initDirs Cannot find specified temporary folder at [/var/lib/tomcat8/temp]
calibre    | qt.qpa.xcb: QXcbConnection: XCB error: 148 (Unknown), sequence: 185, resource id: 0, major code: 140 (Unknown), minor code: 20
calibre    | DBusExport: Failed to connect to DBUS session bus, with error: org.freedesktop.DBus.Error.NotSupported: Using X11 for dbus-daemon autolaunch was disabled at compile time, set your DBUS_SESSION_BUS_ADDRESS instead
calibre    | 17-Oct-2020 13:52:31.115 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Server version:        Apache Tomcat/8.5.39 (Ubuntu)
calibre    | 17-Oct-2020 13:52:31.115 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Server built:          Sep 9 2019 19:47:51 UTC
calibre    | 17-Oct-2020 13:52:31.116 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Server number:         8.5.39.0
calibre    | 17-Oct-2020 13:52:31.116 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log OS Name:               Linux
calibre    | 17-Oct-2020 13:52:31.116 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log OS Version:            5.4.0-48-generic
calibre    | 17-Oct-2020 13:52:31.116 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Architecture:          amd64
calibre    | 17-Oct-2020 13:52:31.117 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Java Home:             /usr/lib/jvm/java-11-openjdk-amd64
calibre    | 17-Oct-2020 13:52:31.117 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log JVM Version:           11.0.8+10-post-Ubuntu-0ubuntu118.04.1
calibre    | 17-Oct-2020 13:52:31.117 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log JVM Vendor:            Ubuntu
calibre    | 17-Oct-2020 13:52:31.117 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log CATALINA_BASE:         /var/lib/tomcat8
calibre    | 17-Oct-2020 13:52:31.117 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log CATALINA_HOME:         /usr/share/tomcat8
calibre    | 17-Oct-2020 13:52:31.118 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: --add-opens=java.base/java.lang=ALL-UNNAMED
calibre    | 17-Oct-2020 13:52:31.118 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: --add-opens=java.base/java.io=ALL-UNNAMED
calibre    | 17-Oct-2020 13:52:31.119 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: --add-opens=java.rmi/sun.rmi.transport=ALL-UNNAMED
calibre    | 17-Oct-2020 13:52:31.119 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djava.util.logging.config.file=/var/lib/tomcat8/conf/logging.properties
calibre    | 17-Oct-2020 13:52:31.119 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager
calibre    | 17-Oct-2020 13:52:31.119 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djdk.tls.ephemeralDHKeySize=2048
calibre    | 17-Oct-2020 13:52:31.120 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djava.protocol.handler.pkgs=org.apache.catalina.webresources
calibre    | 17-Oct-2020 13:52:31.120 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Dorg.apache.catalina.security.SecurityListener.UMASK=0027
calibre    | 17-Oct-2020 13:52:31.120 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Dignore.endorsed.dirs=
calibre    | 17-Oct-2020 13:52:31.120 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Dcatalina.base=/var/lib/tomcat8
calibre    | 17-Oct-2020 13:52:31.121 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Dcatalina.home=/usr/share/tomcat8
calibre    | 17-Oct-2020 13:52:31.121 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djava.io.tmpdir=/var/lib/tomcat8/temp
calibre    | 17-Oct-2020 13:52:31.121 INFO [main] org.apache.catalina.core.AprLifecycleListener.lifecycleEvent The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib]
calibre    | 17-Oct-2020 13:52:31.201 INFO [main] org.apache.coyote.AbstractProtocol.init Initializing ProtocolHandler ["http-nio-8080"]
calibre    | 17-Oct-2020 13:52:31.229 INFO [main] org.apache.tomcat.util.net.NioSelectorPool.getSharedSelector Using a shared selector for servlet write/read
calibre    | 17-Oct-2020 13:52:31.254 INFO [main] org.apache.catalina.startup.Catalina.load Initialization processed in 988 ms
calibre    | 17-Oct-2020 13:52:31.351 INFO [main] org.apache.catalina.core.StandardService.startInternal Starting service [Catalina]
calibre    | 17-Oct-2020 13:52:31.352 INFO [main] org.apache.catalina.core.StandardEngine.startInternal Starting Servlet Engine: Apache Tomcat/8.5.39 (Ubuntu)
calibre    | 17-Oct-2020 13:52:31.401 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployWAR Deploying web application archive [/var/lib/tomcat8/webapps/ROOT.war]
calibre    | Traceback (most recent call last):
calibre    |   File "calibre/gui2/library/views.py", line 972, in database_changed
calibre    |   File "calibre/gui2/library/views.py", line 831, in restore_state
calibre    |   File "calibre/gui2/library/views.py", line 734, in apply_state
calibre    |   File "calibre/gui2/library/views.py", line 689, in apply_sort_history
calibre    |   File "calibre/gui2/library/views.py", line 573, in sort_by_named_field
calibre    |   File "calibre/gui2/library/views.py", line 532, in sort_by_column_and_order
calibre    |   File "calibre/gui2/library/models.py", line 510, in sort
calibre    |   File "calibre/gui2/library/models.py", line 518, in _sort
calibre    |   File "calibre/db/view.py", line 265, in incremental_sort
calibre    |   File "calibre/db/view.py", line 249, in multisort
calibre    |   File "calibre/db/view.py", line 244, in _do_sort
calibre    |   File "calibre/db/cache.py", line 63, in call_func_with_lock
calibre    |   File "calibre/db/cache.py", line 978, in multisort
calibre    | TypeError: '<' not supported between instances of 'str' and 'float'
calibre    | rdpRRGetInfo:
calibre    | QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-abc'
calibre    | qt.qpa.xcb: QXcbConnection: XCB error: 148 (Unknown), sequence: 185, resource id: 0, major code: 140 (Unknown), minor code: 20
calibre    | DBusExport: Failed to connect to DBUS session bus, with error: org.freedesktop.DBus.Error.NotSupported: Using X11 for dbus-daemon autolaunch was disabled at compile time, set your DBUS_SESSION_BUS_ADDRESS instead
calibre    | Traceback (most recent call last):
calibre    |   File "calibre/gui2/library/views.py", line 972, in database_changed
calibre    |   File "calibre/gui2/library/views.py", line 831, in restore_state
calibre    |   File "calibre/gui2/library/views.py", line 734, in apply_state
calibre    |   File "calibre/gui2/library/views.py", line 689, in apply_sort_history
calibre    |   File "calibre/gui2/library/views.py", line 573, in sort_by_named_field
calibre    |   File "calibre/gui2/library/views.py", line 532, in sort_by_column_and_order
calibre    |   File "calibre/gui2/library/models.py", line 510, in sort
calibre    |   File "calibre/gui2/library/models.py", line 518, in _sort
calibre    |   File "calibre/db/view.py", line 265, in incremental_sort
calibre    |   File "calibre/db/view.py", line 249, in multisort
calibre    |   File "calibre/db/view.py", line 244, in _do_sort
calibre    |   File "calibre/db/cache.py", line 63, in call_func_with_lock
calibre    |   File "calibre/db/cache.py", line 978, in multisort
calibre    | TypeError: '<' not supported between instances of 'str' and 'float'
calibre    | 17-Oct-2020 13:52:34.184 INFO [localhost-startStop-1] org.apache.jasper.servlet.TldScanner.scanJars At least one JAR was scanned for TLDs yet contained no TLDs. Enable debug logging for this logger for a complete list of JARs that were scanned but no TLDs were found in them. Skipping unneeded JARs during scanning can improve startup time and JSP compilation time.
calibre    | rdpRRGetInfo:
calibre    | QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-abc'
calibre    | 13:52:34.829 [localhost-startStop-1] INFO  o.a.g.environment.LocalEnvironment - GUACAMOLE_HOME is "/etc/guacamole".
calibre    | 13:52:34.969 [localhost-startStop-1] INFO  o.a.g.rest.auth.HashTokenSessionMap - Sessions will expire after 60 minutes of inactivity.
calibre    | 13:52:35.199 [localhost-startStop-1] INFO  o.a.g.environment.LocalEnvironment - GUACAMOLE_HOME is "/etc/guacamole".
calibre    | 13:52:35.382 [localhost-startStop-1] INFO  o.a.g.t.w.WebSocketTunnelModule - Loading JSR-356 WebSocket support...
calibre    | qt.qpa.xcb: QXcbConnection: XCB error: 148 (Unknown), sequence: 185, resource id: 0, major code: 140 (Unknown), minor code: 20
calibre    | 17-Oct-2020 13:52:35.927 WARNING [localhost-startStop-1] com.google.inject.internal.ProxyFactory.<init> Method [public void org.apache.guacamole.rest.user.UserResource.updateObject(java.lang.Object) throws org.apache.guacamole.GuacamoleException] is synthetic and is being intercepted by [org.apache.guacamole.rest.RESTExceptionWrapper@33333e3b]. This could indicate a bug.  The method may be intercepted twice, or may not be intercepted at all.
calibre    | 17-Oct-2020 13:52:36.126 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.register Registering org.apache.guacamole.rest.extension.ExtensionRESTService as a root resource class
calibre    | 17-Oct-2020 13:52:36.127 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.register Registering org.apache.guacamole.rest.language.LanguageRESTService as a root resource class
calibre    | 17-Oct-2020 13:52:36.127 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.register Registering org.apache.guacamole.rest.patch.PatchRESTService as a root resource class
calibre    | 17-Oct-2020 13:52:36.128 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.register Registering org.apache.guacamole.rest.auth.TokenRESTService as a root resource class
calibre    | 17-Oct-2020 13:52:36.128 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.register Registering org.apache.guacamole.rest.session.SessionRESTService as a root resource class
calibre    | 17-Oct-2020 13:52:36.129 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.register Registering org.codehaus.jackson.jaxrs.JacksonJsonProvider as a provider class
calibre    | 17-Oct-2020 13:52:36.133 INFO [localhost-startStop-1] com.sun.jersey.server.impl.application.WebApplicationImpl._initiate Initiating Jersey application, version 'Jersey: 1.17.1 02/28/2013 12:47 PM'
calibre    | DBusExport: Failed to connect to DBUS session bus, with error: org.freedesktop.DBus.Error.NotSupported: Using X11 for dbus-daemon autolaunch was disabled at compile time, set your DBUS_SESSION_BUS_ADDRESS instead
calibre    | 17-Oct-2020 13:52:36.270 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.getComponentProvider Binding org.codehaus.jackson.jaxrs.JacksonJsonProvider to GuiceManagedComponentProvider with the scope "Singleton"
calibre    | Traceback (most recent call last):
calibre    |   File "calibre/gui2/library/views.py", line 972, in database_changed
calibre    |   File "calibre/gui2/library/views.py", line 831, in restore_state
calibre    |   File "calibre/gui2/library/views.py", line 734, in apply_state
calibre    |   File "calibre/gui2/library/views.py", line 689, in apply_sort_history
calibre    |   File "calibre/gui2/library/views.py", line 573, in sort_by_named_field
calibre    |   File "calibre/gui2/library/views.py", line 532, in sort_by_column_and_order
calibre    |   File "calibre/gui2/library/models.py", line 510, in sort
calibre    |   File "calibre/gui2/library/models.py", line 518, in _sort
calibre    |   File "calibre/db/view.py", line 265, in incremental_sort
calibre    |   File "calibre/db/view.py", line 249, in multisort
calibre    |   File "calibre/db/view.py", line 244, in _do_sort
calibre    |   File "calibre/db/cache.py", line 63, in call_func_with_lock
calibre    |   File "calibre/db/cache.py", line 978, in multisort
calibre    | TypeError: '<' not supported between instances of 'str' and 'float'
calibre    | 17-Oct-2020 13:52:37.059 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.getComponentProvider Binding org.apache.guacamole.rest.extension.ExtensionRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
calibre    | 17-Oct-2020 13:52:37.069 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.getComponentProvider Binding org.apache.guacamole.rest.language.LanguageRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
calibre    | 17-Oct-2020 13:52:37.071 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.getComponentProvider Binding org.apache.guacamole.rest.patch.PatchRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
calibre    | 17-Oct-2020 13:52:37.078 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.getComponentProvider Binding org.apache.guacamole.rest.auth.TokenRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
calibre    | 17-Oct-2020 13:52:37.082 INFO [localhost-startStop-1] com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory.getComponentProvider Binding org.apache.guacamole.rest.session.SessionRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
calibre    | 17-Oct-2020 13:52:37.108 INFO [localhost-startStop-1] org.webjars.servlet.WebjarsServlet.init WebjarsServlet initialization completed
calibre    | 17-Oct-2020 13:52:37.134 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/var/lib/tomcat8/webapps/ROOT.war] has finished in [5,730] ms
calibre    | 17-Oct-2020 13:52:37.141 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-8080"]
calibre    | 17-Oct-2020 13:52:37.171 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in 5916 ms
calibre    | rdpRRGetInfo:
calibre    | QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-abc'
calibre    | qt.qpa.xcb: QXcbConnection: XCB error: 148 (Unknown), sequence: 185, resource id: 0, major code: 140 (Unknown), minor code: 20
calibre    | DBusExport: Failed to connect to DBUS session bus, with error: org.freedesktop.DBus.Error.NotSupported: Using X11 for dbus-daemon autolaunch was disabled at compile time, set your DBUS_SESSION_BUS_ADDRESS instead

(The following keeps repeating over and over until gracefully killed)

calibre    | Traceback (most recent call last):
calibre    |   File "calibre/gui2/library/views.py", line 972, in database_changed
calibre    |   File "calibre/gui2/library/views.py", line 831, in restore_state
calibre    |   File "calibre/gui2/library/views.py", line 734, in apply_state
calibre    |   File "calibre/gui2/library/views.py", line 689, in apply_sort_history
calibre    |   File "calibre/gui2/library/views.py", line 573, in sort_by_named_field
calibre    |   File "calibre/gui2/library/views.py", line 532, in sort_by_column_and_order
calibre    |   File "calibre/gui2/library/models.py", line 510, in sort
calibre    |   File "calibre/gui2/library/models.py", line 518, in _sort
calibre    |   File "calibre/db/view.py", line 265, in incremental_sort
calibre    |   File "calibre/db/view.py", line 249, in multisort
calibre    |   File "calibre/db/view.py", line 244, in _do_sort
calibre    |   File "calibre/db/cache.py", line 63, in call_func_with_lock
calibre    |   File "calibre/db/cache.py", line 978, in multisort
calibre    | TypeError: '<' not supported between instances of 'str' and 'float'
calibre    | rdpRRGetInfo:
calibre    | QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-abc'
calibre    | qt.qpa.xcb: QXcbConnection: XCB error: 148 (Unknown), sequence: 185, resource id: 0, major code: 140 (Unknown), minor code: 20
calibre    | DBusExport: Failed to connect to DBUS session bus, with error: org.freedesktop.DBus.Error.NotSupported: Using X11 for dbus-daemon autolaunch was disabled at compile time, set your DBUS_SESSION_BUS_ADDRESS instead
calibre    | Traceback (most recent call last):
calibre    |   File "calibre/gui2/library/views.py", line 972, in database_changed
calibre    |   File "calibre/gui2/library/views.py", line 831, in restore_state
calibre    |   File "calibre/gui2/library/views.py", line 734, in apply_state
calibre    |   File "calibre/gui2/library/views.py", line 689, in apply_sort_history
calibre    |   File "calibre/gui2/library/views.py", line 573, in sort_by_named_field
calibre    |   File "calibre/gui2/library/views.py", line 532, in sort_by_column_and_order
calibre    |   File "calibre/gui2/library/models.py", line 510, in sort
calibre    |   File "calibre/gui2/library/models.py", line 518, in _sort
calibre    |   File "calibre/db/view.py", line 265, in incremental_sort
calibre    |   File "calibre/db/view.py", line 249, in multisort
calibre    |   File "calibre/db/view.py", line 244, in _do_sort
calibre    |   File "calibre/db/cache.py", line 63, in call_func_with_lock
calibre    |   File "calibre/db/cache.py", line 978, in multisort
calibre    | TypeError: '<' not supported between instances of 'str' and 'float'
calibre    | rdpRRGetInfo:
calibre    | QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-abc'
calibre    | qt.qpa.xcb: QXcbConnection: XCB error: 148 (Unknown), sequence: 185, resource id: 0, major code: 140 (Unknown), minor code: 20
calibre    | DBusExport: Failed to connect to DBUS session bus, with error: org.freedesktop.DBus.Error.NotSupported: Using X11 for dbus-daemon autolaunch was disabled at compile time, set your DBUS_SESSION_BUS_ADDRESS instead

**Comment by Enissay** [comment]
Any fix please ?

I just deployed the latest version `linuxserver/calibre:v5.3.0-ls84` and is still shows the same error: Missing python followed by a bunch of others...

**Comment by aptalca** [comment]
In the original post, calibre seems to crash and is restarted by the supervisor, resulting in the repeated messages.

Perhaps a corrupted database

**Comment by Enissay** [comment]
Actually I am starting a fresh setup. I have no db to start with!

**Comment by j0nnymoe** [comment]
Please could you then provide the same information as the original post? Host os info / docker compose etc

**Comment by ernholm** [comment]
I have to say since it worked flawlessly under the previous version and I did the library check I didn't try just pointing to a new fresh folder for the config which in hindsight of course I should have tried before reporting.

I did now and it started up for me. I then chose to under 4.23 rebuild the database despite no previous signs of corruption and after that updated to 5.3.0 which now worked so your theory about a corrupted database seems to have been the case at least in my instance.

So for me this issue is solved - given there is no significant new information added I'm happy to resolve this and leave similar problems to a new issue.
